### ISSUE TITLE
PHP deprecation errors due to Jetpack_Admin\additional_css_menu()

### ISSUE THREAD

**Issue by todeveni** [Automattic/jetpack#28108]
### Impacted plugin

Jetpack

### Steps to Reproduce

1. Install Jetpack plugin in WordPress with PHP 8.1
2. Load wp-admin as administrator
3. Look at error log

### A clear and concise description of what you expected to happen.

Empty error log

### What actually happened

Two instances of PHP deprecation errors with call stacks.

```
strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated	
  wp-includes/functions.php:7025
wp_is_stream()
  wp-includes/functions.php:2153
wp_normalize_path()
  wp-includes/plugin.php:768
plugin_basename()
  wp-admin/includes/plugin.php:1405
add_submenu_page()
  wp-content/plugins/jetpack/class.jetpack-admin.php:145
Jetpack_Admin::additional_css_menu()
  wp-includes/class-wp-hook.php:308
do_action('admin_menu')
  wp-admin/includes/menu.php:155
```

```
str_replace(): Passing null to parameter #3 ($subject) of type array|string is deprecated	
  wp-includes/functions.php:2160
wp_normalize_path()
  wp-includes/plugin.php:768
plugin_basename()
  wp-admin/includes/plugin.php:1405
add_submenu_page()
  wp-content/plugins/jetpack/class.jetpack-admin.php:145
Jetpack_Admin::additional_css_menu()
  wp-includes/class-wp-hook.php:308
do_action('admin_menu')
  wp-admin/includes/menu.php:155
```

### Browser

Google Chrome/Chromium

### Other information

In [`Jetpack_Admin\additional_css_menu()`](https://github.com/Automattic/jetpack/blob/e980f99a7d41692404318fa9e9d54991b723ac46/projects/plugins/jetpack/class.jetpack-admin.php#L126-L151) you're using `add_submenu_page()` with `null` as first param. According to [WordPress documentation](https://developer.wordpress.org/reference/functions/add_submenu_page/) the first parameter is a required string for parent slug. Since the param is `null` instead of string it's triggering PHP deprecation errors later in the stack.

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by anomiex** [comment]
Changing the `null` to empty string should work the same, since the `plugin_basename()` call winds up turning it into that anyway for all the subsequent code in the function.

**Comment by todeveni** [comment]
When is this supposed to land on an actual Jetpack release? Should this be milestoned?

**Comment by anomiex** [comment]
Looks like it was included in 11.8, which was released yesterday.
